pkg/validation/validation.go

```diff
@@ -13,9 +13,6 @@ import (
 	"github.com/operator-framework/api/pkg/validation/internal"
 )
 
-// PackageManifestValidator implements Validator to validate package manifests.
-var PackageManifestValidator = internal.PackageManifestValidator
```

Comment:
I assume this will be moved to operator-registry, like the bundle validator was? The SDK uses this validator but not the `PackageUpdateGraphValidator`.